src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i=0;i<3;i++) {
+            String randomBall = new Lottery().getRandomBall();
```

Comment:
no need to create a new lottery object every time, take it out to a variable before the loop and use it in the loop